shared/charts/gpii-flushtokens/templates/job.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
```

Comment:
Please rename this file to cronjob.yaml.

Reply:
Done